downstream/_auxlib/_unpack_hex.py

```diff
@@ -36,7 +44,8 @@ def unpack_hex(hex_str: str, num_items: int) -> np.ndarray:
         # handle 4-bit values by processing ascii ordinals directly
         ascii_codes = np.fromstring(hex_str.lower(), dtype="S1", sep="").view(
             np.uint8,
-        )
+        ).byteswap()
```

Comment:
Ah you're right, it is! Since the test for the 4bit case involved a symmetric number (`0xF0F0`), this wasn't caught -- the tests fail when changing this to `0xA0F0`.